inlong-manager/manager-web/sql/apache_inlong_manager.sql

```diff
@@ -759,8 +759,8 @@ CREATE TABLE IF NOT EXISTS `group_heartbeat`
     `component`        varchar(64)  NOT NULL DEFAULT '' COMMENT 'Component name, such as: Agent, Sort...',
     `instance`         varchar(64)  NOT NULL DEFAULT '' COMMENT 'Component instance, can be ip, name...',
     `inlong_group_id`  varchar(256) NOT NULL DEFAULT '' COMMENT 'Owning inlong group id',
-    `status_heartbeat` text         DEFAULT NULL COMMENT 'Status heartbeat info',
-    `metric_heartbeat` text         DEFAULT NULL COMMENT 'Metric heartbeat info',
+    `status_heartbeat` text                  DEFAULT NULL COMMENT 'Status heartbeat info',
```

Comment:
Why is there indentation here, the previous format should be more standardized

Reply:
1. It's the IDEA auto-formated.
2. The new format looks better: `DEFAULT` is in the same column.